tests/integration/test_multiple.py

```diff
@@ -17,7 +17,7 @@
                     # using RPM to provide gomod and npm in the image
                     {"type": "rpm"},
                 ),
-                flags=["--dev-package-managers"],
+                flags=[],
```

Comment:
Again, this should have been part of a standalone commit I mentioned elsewhere in this review, this was an oversight during my RPM promotion work.